src/webots/nodes/WbIndexedLineSet.cpp

```diff
@@ -144,6 +145,8 @@ void WbIndexedLineSet::reset() {
 
 int WbIndexedLineSet::computeCoordsData(float *data) {
   WbMFInt::Iterator it(*mCoordIndex);
+  if (it.hasNext() == 0)
```

Comment:
```suggestion
  if (!it.hasNext())
```

This looks simpler / easier to understand to me.

Reply:
Definitely, thanks for your update